website/content/en/docs/concepts/scheduling.md

```diff
@@ -602,4 +602,6 @@ topologySpreadConstraints:
 - maxSkew: 1
   topologyKey: capacity-spread
   whenUnsatisfiable: DoNotSchedule
+  labelSelector:
```

Comment:
Are there any other places that we should specify this labelSelector throughout our docs?

Reply:
For `topologySpreadConstraints` at least this looks to be the only place.